test/Esprima.Tests.Test262/Esprima.Tests.Test262.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <OutputType>Exe</OutputType>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Spectre.Console" Version="0.42.0" />
```

Comment:
See that you decided to have some fun too ;)

Reply:
There has to be some incentive to do such laborous tasks! A fun experiment to get red/green status on screen without worrying about creating new dependencies for consumers at least...

Reply:
It's kinda cool though:

![image](https://user-images.githubusercontent.com/171892/140618516-a369b2ec-4e00-4c60-94d5-94d0fa46a165.png)

Reply:
Is that just locally or does it work also in the GH console?

Reply:
It should revert to least cool experience on non-interactive console, experimenting here...